docs/components/configurationPanel.md

```diff
@@ -219,6 +227,8 @@ unchanged, then the buttons will be disabled. Otherwise, the buttons are enabled
 
 - Returns: Nothing.
 
-Stores all the unsaved changes in Chrome's storage when the value of any configuration option is changed. These values
-are saved separately without applying the gamepad configuration and are used to prevent loss of unsaved configuration
-due to the panel getting closed accidentally. These values are restored when the panel is reopened.
+Stores all the unsaved changes in Chrome's storage (using
+[`chrome.storage`](https://developer.chrome.com/extensions/storage) API) when the value of any configuration option is
+changed. These values are saved separately without applying the gamepad configuration and are used to prevent loss of
```

Comment:
I'd avoid using "saved" for quite so many things here, i.e. you might also mean saving unsaved changes rather than preserving the values in the model.  Something like "Unsaved changes are preserved to prevent losing them when the configuration panel is closed" sounds better.